verification-client/src/resource.rs

```diff
@@ -158,7 +161,7 @@ impl VerificationClientResource {
         let content_type = response
```

Comment:
a content type should always be set, we should fail here if its not. It should also make the rest of the code cleaner

Reply:
not that much but can unwrap the optional as well

Reply:
we do not send content-type in 204 responses